bcdaworker/queueing/manager/alr.go

```diff
@@ -118,6 +115,9 @@ func (q *masterQueue) startAlrJob(job *que.Job) error {
 	}
 	// End of validation
 
+	// Check if the job was cancelled
+	go checkIfCancelled(ctx, q, cancel, jobArgs)
+
```

Comment:
I'm thinking we should check if the job is cancelled first before spinning up a go routine. This way, we don't pointlessly spin one up when the job was cancelled.

Reply:
👍 Makes sense to do all that validation first!